src/feature_scaling.jl

```diff
@@ -127,14 +127,14 @@ function rescale!(X::AbstractMatrix, μ::AbstractVector, σ::AbstractVector, ::O
     μ, σ
 end
 
-function rescale!(X::AbstractVector, μ::AbstractVector, σ::AbstractVector, ::ObsDim.Constant{1})
```

Comment:
Well, since `X` is a vector, the fallback method you implemented transforms `ObsDim.Last` to `ObsDim.Constant{1}` anyway.

I think here `::ObsDim.Constant{1}` was cleaner than `::ObsDim.Constant` (which would also allow `ObsDim.Constant{2}()`), because if the data `X` is a vector, then `1` is the only sensible dimension to choose. i.e. I'd expect a method error if I use `ObsDim.Constant{2}()`.